cfme/fixtures/provider.py

```diff
@@ -395,6 +395,106 @@ def console_template_modscope(provider):
     return _get_template(provider, 'console_template')
 
 
+@pytest.fixture(scope="function")
```

Comment:
Required: use @pytest.fixture
and add _funcscope to the fixture name to make it more clear

Reply:
no changes needed, as discussed in irc